lib/Data/Validate/Sanctions.pm

```diff
@@ -41,7 +44,7 @@ sub update_data {
 
     my $updated;
     foreach my $k (keys %$new_data) {
-        if (ref($self->{_data}{$k}) ne 'HASH' || $self->{_data}{$k}{updated} < $new_data->{$k}{updated}) {
+        if (!$self->{_data}{$k} || $self->{_data}{$k} ne 'HASH' || Compare($self->{_data}{$k}, $new_data->{$k})) {
```

Comment:
if `$self->{_data}{$k}` , `$new_data->{$k}` is same, then we update it ?
I guess this logic should be reversed.

Reply:
You are right. I'll correct it.

Reply:
Fixed.